templates/invoice.html

```diff
@@ -0,0 +1,343 @@
+{% load price from taxed_prices %}
+<!DOCTYPE html>
+<html>
+
+<head>
+    <style>
+        @page {
+            background-color: #FAFAFA;
+            margin: 0.5cm;
+        }
+
+        .section-header,
+        .section-invoice-info {
+            background-color: #EFF5F8;
+            height: 235px;
+        }
+
+        .section-left {
+            width: 50%;
+            float: left;
+        }
+
+        .section-right {
+            width: 50%;
+            float: right;
+        }
+
+        .header-category {
+            font-family: Inter;
```

Comment:
According to the design, we should also use this font for the body of the invoice.